client/src/main/scala/io/delta/sharing/client/DeltaSharingClient.scala

```diff
@@ -495,7 +500,9 @@ class DeltaSharingRestClient(
       val (version, respondedFormat, lines, _) = getFilesByPage(table, target, request)
       (version, respondedFormat, lines)
     } else {
-      val response = getNDJson(target, request)
+      val response = getNDJsonPost(
+        target = target, data = request, setIncludeEndStreamAction = true
```

Comment:
(optional, for my readability)
consider renaming: includeEndStreamAction -> endStreamActionEnabled
We can then setIncludeEndStreamAction = endStreamActionEnabled where we are setting it to true, and then at other places only test for setIncludeEndStreamAction.

Reply:
do you mean the renaming of the header or variable name?

Reply:
@chakankardb

Reply:
The variable name @linzhou-db

Reply:
updated